src/ralph/back_office/admin.py

```diff
@@ -108,8 +109,8 @@ class BackOfficeAssetAdmin(
         'user', 'owner', 'user__segment', 'user__company', 'user__department',
         'user__employee_id', 'property_of', 'invoice_no', 'invoice_date',
         'order_no', 'provider', 'budget_info', 'depreciation_rate',
-        'depreciation_end_date', 'force_depreciation', LiquidatedStatusFilter,
-        TagsListFilter
+        'depreciation_end_date', 'force_depreciation', 'buyout_date',
```

Comment:
`buyout_date` filter should filter out assets which model's category has `show_buyout_date` set to `False`